opus/application/static_media/js/browse.js

```diff
@@ -8,6 +8,13 @@
 /* globals default_pages, default_columns */
 
 /* jshint varstmt: false */
+// font awesome icon class
```

Comment:
The /* jshint */ line needs to go after your const lines and right before the "var".